config/openiotsdk/CMakeLists.txt

```diff
@@ -292,9 +292,3 @@ endif()
 target_include_directories(openiotsdk-chip INTERFACE
     ${CHIP_ROOT}/config/openiotsdk/mbedtls
 )
-
-if (NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Debug")
```

Comment:
Is the goal to keep assertions in release builds ?

Reply:
The SDK release build provides this definition. We don't need it anymore.